### ISSUE TITLE
Make it easy to get the ssh connection parameters inside browser editor

### ISSUE THREAD

**Issue by josh-braid** [gitpod-io/gitpod#9770]
## Is your feature request related to a problem? Please describe
Often, I open the gitpod editor and want to copy the ssh connection line, but it's not in the hamburger menu, so I have to open a new tab to go to my workspaces and copy the link.

## Describe the behaviour you'd like
I'd like to be able to get the link from inside the gitpod window, maybe in the hamburger menu next to the "Open in VSCode" options.

## Describe alternatives you've considered

Alternate 1: it'd be nice if opening a gitpod workspace opened it in a new window, rather than replacing the window. Since vscode is such a heavyweight app that loads slowly, I'm not inclined to just hit back, so I find myself opening a new tab a lot just to go back to the workspace list.

Alternate 2 (true dream solution): If I could specify my terminal emulator in the gitpod vscode plugin and it could run the gitpod tasks in tabs of my emulator (automatically connecting over ssh first) rather than running them in the vscode terminal (which is awful).

## Additional context
My current workflow at the beginning of the day is like:
1. Open gitpod in the browser, wait for it to completely load (otherwise opening local vscode often fails to connect)
2. Once it's up, go to the hamburger menu and select "open in vscode"
3. One or two rounds of "Could not connect, retry or close remote?"
4. Close all the vscode terminal task windows.
5. Open a new tab with gitpod workspaces in it, copy the ssh connection params
6. For each tab:
    - open a new tab in my terminal emulator for each task (currently Konsole) and paste connection params
    - cd to the `/workspace/<reponame>` since ssh lands me at the home directory
    - start the task for that tab (typescript watch or webpack watch etc)
7. Close the gitpod browser window
8. Make sure not to leave the computer alone for too long because otherwise vscode / ssh will disconnect and I'll have to do it over again.
9. Install github copilot in the remote vscode window (vscode plugin not available in the browser)

**Comment by loujaybee** [comment]
Hey @josh-braid ! Thanks so much for the feedback, and the detailed write-up. Hearing the steps you take to get setup was also really helpful. We don't have a well optimised workflow for users who are using or connecting from external terminals, however I've got some thoughts on your workflow that could help you out a little...

> Open gitpod in the browser, wait for it to completely load (otherwise opening local vscode often fails to connect)

Have you considered setting VS Code Desktop as a user preference, then when the workspace loads, you'll get a splash page with a prompt... might save a little time. You can also get the SSH copy/paste from this screen. 

A local CLI open command would be nice here (working on it! #7185)

> Once it's up, go to the hamburger menu and select "open in vscode"

Small optimisation... but you can also use command palette here to save clicking (if that's your thing!)

> One or two rounds of "Could not connect, retry or close remote?"

SSH method of connection is being updated quite soon, should bring some significant connection stability benefits https://github.com/gitpod-io/gitpod/issues/8513

> cd to the /workspace/<reponame> since ssh lands me at the home directory

Agree, this needs an update - https://github.com/gitpod-io/gitpod/issues/9377

> start the task for that tab (typescript watch or webpack watch etc)

You can use [`gp tasks`](https://www.gitpod.io/docs/command-line-interface#tasks) to fetch up any running processes defined in your .gitpod.yml (could help) 

> Make sure not to leave the computer alone for too long because otherwise vscode / ssh will disconnect and I'll have to do it over again.

https://github.com/gitpod-io/gitpod/issues/9038 🙏

**Comment by josh-braid** [comment]
Hey @loujaybee thanks for the response!

> Have you considered setting VS Code Desktop as a user preference, then when the workspace loads, you'll get a splash page with a prompt... might save a little time. You can also get the SSH copy/paste from this screen.

I found this option, but when I switch to it, I get the message "container workspace completed; containers of a workspace pod are not supposed to do that" and it fails to launch. I just tried it with workspace `braidlending-braid-0t2yvtpx9aj` if that helps diagnose it.

Thanks for pointing out gp tasks, I didn't know about that
